### DISCUSSION TITLE
onClose handler of context modal props does not receive cancel flag

### DISCUSSION THREAD

**Discussion by lukasbash** [mantinedev/mantine#5582] [Q&A]
### Dependencies check up

- [X] I have verified that I use latest version of all @mantine/* packages

### What version of @mantine/* packages do you have in package.json?

7.4.0

### What package has an issue?

@mantine/core

### What framework do you use?

Vite

### In which browsers you can reproduce the issue?

All

### Describe the bug

The current signature of the `onClose` handler of the context modal props is defined as `onClose?: (() => void) | undefined`.
By using the `closeContextModal` method of a modal context, you can pass an optional cancel flag as a boolean.

This flag should be passed to the close handler too, to further decide on possible callbacks when closing a modal.

### Self-service

- [ ] I would be willing to implement a fix for this issue

**Comment by wagnergerald** [comment]
Any update on this one?

**Comment by mariohirzi** [comment]
@rtivital Any update ?
